themes/sensei-learning-mode/templates/lesson.4-0-2.html

```diff
@@ -0,0 +1,60 @@
+<!-- wp:group {"className":"sensei-learning-mode__header sensei-learning-mode__frame sensei-version--4-0-2", "tagName": "header"} -->
```

Comment:
This file represents what users who remain on the '4.0.2' template version (because they have customizations) will have. So we shouldn't change it, since we won't change what they are running on their site either.

I'm afraid that means they will have a template with all the `sensei-course-theme__*` classes. Which brings us to the same issue as the blocks, with these options:

- Have both `sensei-learning-mode` and `sensei-course-theme` prefix in the CSS
- Completely copy over all CSS to the `4-0-2` folder, and keep the course-theme prefix for those
- Don't rename the classes

Sorry that I didn't think about this sooner, but probably option 3 is the easiest here too.

cc @Automattic/zen would it be too much disconnect when working with the code to keep the `sensei-course-theme__...` classnames?

Reply:
I personally don't think it's much of an issue 👍